cmd/internal/collector/collector.go

```diff
@@ -97,7 +98,13 @@ func (cc CCloudCollector) handleResponse(response QueryResponse, ccmetric CCloud
 		}
 
 		if topicPresent && clusterPresent && rule.ShouldIgnoreResultForRule(topic, cluster, ccmetric.metric.Name) {
-			continue
+			continue METRICSLOOP
+		}
+
+		for _, currentRegex := range rule.ExcludeTopicsRegex {
```

Comment:
Filtering on the exporter side could be dangerous, mostly because we could reach the limits of the 1000 points without even notifying it. Maybe we could add a `warning` in case we receive 1000 points?

Reply:
Hmmm I'm confused here... The way I understand it is that there isn't really a data point limit thrown when querying.
So a query can receive 1000+ data points.
Mind explaining a bit more?

Reply:
Actually, there is a limit, a query can not receive more than 1000+ data points.
If there is more than 1000 point to return, only the "first" 1000 points will be returned, the rest will be ignored.

Some peoples encountered issues where they could only partially see the results of their query due to that (e.g. when grouping per partition and you have a lot of partition....).
That's why I introduced a limit that we need to filter per topics if you want to group by partition.

Reply:
If you try to set a limit greater than 1000, you get:
```{
    "errors": [
        {
            "detail": "limit must be less than or equal to 1000",
            "status": "422"
        }
    ]
}
```

Reply:
Ok, I re read the docs and your comments like 5 times now and I think I get it, thanks :) I'll add the warning.